Leanplum/LPRequestStoring.h

```diff
@@ -9,27 +9,25 @@
 #ifndef LPRequestStoring_h
```

Comment:
do we need this ifndef? can we please remove it.

Reply:
this helps with preventing circular imports (the issue that we faced when `Leanplum.h` was importing files that were importing `Leanplum.h`